### ISSUE TITLE
Target IP clarification

### ISSUE THREAD

**Issue by discort** [alex/what-happens-when#271]
Needs a clarification how to get the target IP address in `ARP process` section.

**Comment by mallika-b** [comment]
Yeah, it's not the Target IP that's needed, it's the Target MAC address.
This pull request https://github.com/alex/what-happens-when/pull/304 needs to be merged

If you look at the ARP Request snippet, it shows that in the beginning, there is a target IP address, but the target MAC address is not known, that is why they're using the broadcast MAC address. 

At the end, in the ARP Reply snippet, you see the Request Sender Mac and IP are now in the Target fields, the Request Target IP is now the Sender IP - and the MAC information associated with that IP (which can be cached in the ARP for future use) is now available as Sender MAC address.
